grassroots-backend/package.json

```diff
@@ -53,6 +54,7 @@
     "@types/node": "^22.15.3",
     "@types/passport": "^1.0.17",
     "@types/passport-local": "^1.0.38",
+    "@types/passport-openidconnect": "^0.1.3",
```

Comment:
passport-google-oicd is untyped, but the types are mostly from here, so I just used these.